lib/reamaze_api/client.rb

```diff
@@ -37,15 +37,17 @@ def on_complete(response)
 
     # Public: Initialize a new Client instance.
     #
-    # brand: Brand name (subdomain from your Reamaze URL)
-    # login: Reamaze login
-    # token: Reamaze API token
+    # brand:      Brand name (subdomain from your Reamaze URL)
+    # login:      Reamaze login
+    # token:      Reamaze API token
+    # exceptions: If true, raise ReamazeAPI::Error for HTTP 400-599
     #
     # Yields a Faraday::Connection if a block is given.
     #
     # Returns nothing.
-    def initialize(brand:, login:, token:)
-      @url = URI.parse(API_URL % { brand: brand })
+    def initialize(brand:, login:, token:, exceptions: false)
```

Comment:
non-blocking style note, i'd lean towards `raise_exceptions` for this; otherwise would expect `@exceptions` to be a collection of exceptions.